### ISSUE TITLE
In android 10 , NormalFilePickActivity not working

### ISSUE THREAD

**Issue by harleendutt5** [fishwjy/MultiType-FilePicker#63]
I'm using NormalFilePickActivity to pick doc and pdf etc. type file from my device, it is working fine in all versions except android 10.  
In android 10, it does not show files , but if i use NormalFilePickActivity to get images, then it is working fine.
Thanks in advance.

**Comment by paul-castro** [comment]
any updates on this issue?

**Comment by harleendutt5** [comment]
@paul-castro 
Yes, in android 10 there is updation in storage access , so i need to add below line in manifest, and it works fine

requestLegacyExternalStorage=true

**Comment by rvats011990** [comment]
requestLegacyExternalStorage=true is also not working in my case. The file picker is coming empty.
 Intent intent4 = new Intent(activity, NormalFilePickActivity.class);
 intent4.putExtra(Constant.MAX_NUMBER, 1);
 intent4.putExtra(NormalFilePickActivity.SUFFIX, new String[] {"doc", "docx","txt", "pdf"});
 startActivityForResult(intent4, REQUEST_CODE_Word_Doc);

android:requestLegacyExternalStorage="true"
added in mainfest.

Could you please help me with the same...
